### PR TITLE
Refactor frontend

### PR DESCRIPTION
**Pull request by fredpetersen**

### CODE REVIEW

**Review by lucasfth — CHANGES_REQUESTED** [review]
lgtm when the weather forecast has been 🗡️

File: NEWgIT.Client/Pages/Counter.razor [hunk 1]
```diff
@@ -0,0 +1,18 @@
+﻿@page "/counter"
```
Comment (Fefa1999) [inline NEWgIT.Client/Pages/Counter.razor:1]: 🗡️

**Review by lucasfth — APPROVED** [review]
etydsetiiilgtm

**Review by ChrBank — APPROVED** [review]
lgtm

**Review by duckth — APPROVED** [review]

**Pull request merged**